.circleci/config.yml

```diff
@@ -0,0 +1 @@
+version: 2
```

Comment:
I'm not sure it requires more settings, we'll see once it is enabled.
This is just for running `standard` on the 2 js files we have.
There are no tests so testing multiple node versions makes no sense.